components/Customer/CustomerForm/index.js

```diff
@@ -0,0 +1,44 @@
+import styled from "styled-components";
+
+const FormContainer = styled.fieldset`
```

Comment:
A `fieldset` is part of a form, but not the container of a form. I see what you are going to do, but rename this styled component to `StyledFormFieldset` or similar to be more specific.